src/user-research/choosing-and-using-research-methods.md

```diff
@@ -123,18 +125,18 @@ Choose personas when:
 
 Interactive workshops with small groups can be an effective method for user research.
 
-They can help you learn more about the things that actual or likely users do, how they do them, how they think and make decisions, and how they feel about their experiences.
+They can help us learn more about the things that actual or likely users do, how they do them, how they think and make decisions, and how they feel about their experiences.
 
-Choose workshops when you want to:
+We choose workshops when we want to:
 
 * see how different people, such as colleagues in different roles, or parents and children, work together to make a decision or to get something done
 * get a more detailed understanding of people who’ve had a similar experience
 
-You can also do workshops with stakeholders, teams and subject matter experts, as a good first step to get an overview of a new area and build trust in preparation for more detailed research.
+We also do workshops with stakeholders, teams and subject matter experts, as a good first step to get an overview of a new area and build trust in preparation for more detailed research.
 
-Group workshops may not be appropriate when researching very sensitive subjects, or with people in vulnerable contexts, who may want more privacy and confidentiality.
+Group workshops may not be appropriate when [researching very sensitive subjects](/user-research/doing-research-safely/), or with people in vulnerable contexts, who may want more privacy and confidentiality.
```

Comment:
Only true when one does not have control over who is recruited/included for that workshop.

Not 100% true if the workshop recruits people with those specific experiences on those specific sensitive topics.
I have ran and been part of workshops/focus groups on disability and chronic illness and they have worked well because all people in the workshop feel it is a safe space -because they know explicitly that others joining the same workshop have a shared experience.

Reply:
True, and have had the same experience.
Maybe focus on the participant preference.

When [researching very sensitive subjects](/user-research/doing-research-safely/), we understand that some participants may want more privacy and confidentiality than a group workshop can provide.

Reply:
yes, it agree that this is better "When [researching very sensitive subjects](https://github.com/user-research/doing-research-safely/), we understand that some participants may want more privacy and confidentiality than a group workshop can provide."